### DISCUSSION TITLE
Unable to create repo with name same as my username

### DISCUSSION THREAD

**Discussion by akshitsangwan** [community/community#37086] [Other Feature Feedback, Questions, & Ideas]
Hey there, I am not able to update my username's readme file. Whenever I do, it shows up with a warning- "The repository akshitsangwan has been retired and cannot be reused".
Please help me solve this issue.
![Screenshot (67)](https://user-images.githubusercontent.com/85335888/197476905-a12ac248-9b5b-4ffe-969b-79faa80573dd.png)

**Comment by DSDmark** [comment]
Hey @akshitsangwan,

### about the error

> ```the repository tiyea has been retired and connot be reused```

### if that is retired, then I doubt you change your username very quickly, it tends to retire repos

Here is the same discussion about the error "The repository [name] has been retired and cannot be reused"
The repository [name] has been retired and cannot be reused:-

 > https://github.com/orgs/community/discussions/23480

check this out: https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement

**Comment by ossanna16** [comment]
Hi there @akshitsangwan and welcome to our community! Thank you for asking a great question 🙂
